lib/taiko.js

```diff
@@ -381,6 +381,25 @@ module.exports.title = async () => {
     return result.result.value;
 };
 
+/**
+ * Sets a cookie with the given cookie data; may overwrite equivalent cookie if they exist.
+ *
+ * @example
+ * setCookie("CSRFToken","csrfToken", {url: "http://the-internet.herokuapp.com"})
+ * 
+ * @param {string} name - Cookie name.
+ * @param {string} value - Cookie value.
+ * @returns {Promise<Object>} - Object with the description of the action performed.
+ */
+module.exports.setCookie = async (name, value, options = {}) => {
+    validate();
+    if (options.url === undefined && options.domain === undefined) throw new Error('Atleast URL or Domain needs to be specified for setting cookies');
```

Comment:
> Atleast URL or Domain needs to be specified for setting cookies

can this be

Please specify a domain or URL for setting cookies

Reply:
Done